app/__init__.py

```diff
@@ -0,0 +1,11 @@
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+import os
+import app.views
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///test.db'
```

Comment:
You can work with sqlite, but if you're going to work with postgres you should edit connection string